.github/workflows/spelling.yml

```diff
@@ -65,7 +65,7 @@ jobs:
     outputs:
       followup: ${{ steps.spelling.outputs.followup }}
     runs-on: ubuntu-latest
-    if: (contains(github.event_name, 'pull_request') && github.head_ref != 'l10n_dev') || github.event_name == 'push'
+    if: (contains(github.event_name, 'pull_request') && github.head_ref != 'l10n_dev')
```

Comment:
How does this pull request event work, would it only get triggered when a pr is created? If the case then it probably wouldn't be triggered again by subsequent pushes to the pr.

I wonder maybe a better way is to just  github.head_ref != dev && github.head_ref != master, so it doesn't check pushes to dev and master branches.

Reply:
It is also triggered on synchronize event (any push to the pr head triggers it).

Reply:
simply ignoring dev or master is enough though. but if you push to a fork the check is also triggered. I want to just check in the pr.

Reply:
oh forget to comment on push at the beginning of the file